modules/serverless-creating-broker-admin.adoc

```diff
@@ -3,17 +3,22 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker-admin_{context}"]
-= Creating a broker for your namespace automatically using namespace annotation
+= Creating a broker automatically using namespace annotation
 
 If you have cluster administrator permissions, you can create a broker automatically by annotating a namespace.
 
 .Prerequisites
 * Knative Eventing installed.
 * Cluster administrator permissions for {product-title}.
 
+[NOTE]
+====
+If you do not have cluster administrator permissions, you must create the broker using the manual method outlined in xref:../modules/serverless-creating-broker.adoc#serverless-creating-broker[Creating a broker manually].
```

Comment:
We cannot include xrefs in modules 🙁

Reply:
OOF, forgot about this. No worries, I can remove it since I mention this in the assembly now anyway. Good catch and thank you!